opentelemetry-auto-instrumentation/opentelemetry-auto-instrumentation.gemspec

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'version'
+
+Gem::Specification.new do |spec|
+  spec.name        = 'opentelemetry-auto-instrumentation'
+  spec.version     = VERSION
+  spec.authors     = ['OpenTelemetry Authors']
+  spec.email       = ['cncf-opentelemetry-contributors@lists.cncf.io']
+
+  spec.summary     = 'auto-instrumentation for opentelemetry ruby'
+  spec.description = 'auto-instrumentation for opentelemetry ruby'
+  spec.homepage    = 'https://github.com/open-telemetry/opentelemetry-ruby-contrib'
+  spec.license     = 'Apache-2.0'
+
+  spec.files = Dir.glob('lib/**/*.rb') +
+               Dir.glob('*.md') +
+               ['LICENSE']
+
+  spec.require_paths = ['lib']
+  spec.required_ruby_version = '>= 3.1'
```

Comment:
simiilar question mentioned above regarding support for Ruby 3.1

Reply:
Updated